setup.cfg

```diff
@@ -50,3 +50,16 @@ exclude=migrations,conf.py,.tox,.git,pootle/static/js,pootle/translations,pootle
 # Default ignore list for ref E121,E123,E126,E226,E24,E704
 ignore=E129,E226,E402,W503
 statistics=True
+
+[flake8]
+# See [pep8] above, [pep8] is the reference, please keep in sync.
+# You would think flake8 would have the decency of reading the pep8 configs,
+# but it doesn't so we have to repeat them.
+max-line-length=84
+exclude=migrations,conf.py,.tox,.git,pootle/static/js,pootle/translations,pootle/locale,pootle/assets
```

Comment:
Use just `pootle/static` to also exclude CSS.

Perhaps `pootle/dbs`, `pootle/exports`, `pootle/log`, `pootle/templates` and `pootle/tools` can also be excluded.